rakudup

```diff
@@ -1,17 +1,16 @@
 #!/usr/bin/env bash
 # https://github.com/stmuk/rakudup
 
-RAKUDO_ROOT="${HOME}/.rakudup"
+RAKUDO_ROOT=$HOME/.rakudup
 
 set -e
 
-if [ ! -z "$DEBUG" ]
-then
-    set -x 
+if [[ $DEBUG ]]; then
+    set -x
```

Comment:
This breaks DEBUG=0

Reply:
You sure it does?

Reply:
Yes! I think it would have to be [ $DEBUG ] since [[ 0 ]] is [ "0" ] which is true.

I now remember why I use Perl! grrrr

Reply:
```
DEBUG=0
if [[ $DEBUG ]]; then echo foo; fi       # foo
if [ ! -z "$DEBUG" ]; then echo bar; fi  # bar
```
Looks identical to me